src/_posts/platform/databases/2000-01-01-maintenance-windows.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Database Maintenance Windows
+nav: Maintenance Windows
+modified_at: 2023-08-31 00:00:00
+tags: databases maintenance
+index: 0
+---
+
+Database Maintenance Windows is our system tailored to schedule the maintenances needed on your databases. It allows you to keep them secure and up-to-date, while being aware of the time period where your database could be temporarily unavailable.
```

Comment:
I'd simplify this sentence, the more is simple the more it is understandable.
In addition, "It allows you to keep them secure..", wouldn't be "It allows us to keep them.." instead as we are the ones executing the update/operation/... during maintenance.

Reply:
You are right! Fixing